resources/default-config.kdl

```diff
@@ -65,6 +65,10 @@ input {
     // Scale is a floating-point number, but at the moment only integer values work.
     scale 2.0
 
+    // Transform allows rotating the output, values are:
+    // Normal, 90, 180, 270, Flipped, Flipped90, Flipped180 and Flipped270
```

Comment:
To be consistent with the rest of the config, make these kebab case, i.e. "normal", "flipped-90".